src/main/java/org/sonar/ucfg/Expression.java

```diff
@@ -101,4 +108,40 @@ public boolean isConstant() {
       return true;
     }
   }
+
+  class Clazz implements Expression {
```

Comment:
Why not using `ClassName` ? that's the name used in the JIRA ticket and specification.